### PR TITLE
Add net6.0 target framework, bump editor

### PR DESCRIPTION
**Pull request by sandyarmstrong**
Reference MS.VS.Language explicitly to work around restore bug.

### CODE REVIEW

**Review by mrward — APPROVED** [review]

File: Editor/MonoDevelop.Xml.Editor.csproj [hunk 1]
```diff
@@ -19,14 +19,23 @@
...
+    <PackageReference Include="Microsoft.VisualStudio.Language.StandardClassification" Version="17.1.387" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.1.46" ExcludeAssets="runtime" />
   </ItemGroup>
 
   <ItemGroup>
     <Reference Include="System.ComponentModel.Composition" />
   </ItemGroup>
 
+  <!-- Turn off windows dependency check. Some packages still have windows-specific dependencies, even though in the cases that matter they are not used at run-time -->
+  <Target Name="IgnoreWindowsReferenceError" BeforeTargets="_CheckForTransitiveWindowsDesktopDependencies">
```
Comment (sandyarmstrong) [inline Editor/MonoDevelop.Xml.Editor.csproj:34]: This target should only be necessary when a `net6.0*` project references `net472` nupkgs. In this project, every package _should_ resolve as `netstandard2.0` for the `net6.0` target framework, but for some reason NuGet is resolving them as `net472`, triggering this error and also https://github.com/NuGet/Home/issues/5957.

Comment (sandyarmstrong) [reply Editor/MonoDevelop.Xml.Editor.csproj:34]: I should note that this happens even if I remove the `net472` from `TargetFrameworks` completely.

**Review by KirillOsenkov — APPROVED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -30,13 +30,13 @@ jobs:
     - name: Build
       run: dotnet build MonoDevelop.Xml.sln -c ${{ matrix.config }} --no-restore
 
-    - name: Test (Windows)
-      if: startsWith (matrix.os, 'windows')
-      run: dotnet test -c ${{ matrix.config }} --no-build
+    # - name: Test (Windows)
```
Comment (sandyarmstrong) [inline .github/workflows/main.yml:33]: Tests won't work unless minieditor gets updated.

Comment (KirillOsenkov) [reply .github/workflows/main.yml:33]: we don’t care at this point

**Comment by KirillOsenkov** [comment]
Can you do me a favor and merge/bump? thanks. Let me know if I can help with any tasks.

**Pull request merged**